synapse/cortex.py

```diff
@@ -939,6 +939,8 @@ async def initServiceStorage(self):
     async def initServiceRuntime(self):
 
         # do any post-nexus initialization here...
+        if self.isactive:
```

Comment:
Did we learn something that makes us think we should only do this to an active cortex?

Reply:
Yeah, a mirror that isn't caught up shouldn't update the nexuslog position.